modules/layers/src/column-layer/stroke-truncated-cone-geometry.js

```diff
@@ -0,0 +1,121 @@
+import {Geometry, uid} from '@luma.gl/core';
+
+const INDEX_OFFSETS = {
+  x: [2, 0, 1],
+  y: [0, 1, 2],
+  z: [1, 2, 0]
+};
+
+export default class TruncatedConeGeometry extends Geometry {
```

Comment:
Nit: `class StrokeTruncatedConeGeometry`?

Not possible to inherit from https://github.com/uber/luma.gl/blob/master/modules/core/src/geometries/truncated-cone-geometry.js

and just transform/change some of the results?

Reply:
Or `ColumnOutlineGeometry`, `ColumnWireframeGeometry` etc?